dbfit-java/build.gradle

```diff
@@ -83,7 +83,10 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task dbConnectionProperties {
```

Comment:
Not really. `connectionProperties` are now really just the values from the properties file. May be renaming `connectionProperties` to `connectionFileProperties` would be enough to distinguish them.